code/math/vecmat.cpp

```diff
@@ -2837,3 +2848,9 @@ void vm_interpolate_angles_quick(angles *dest0, angles *src0, angles *src1, floa
 	dest0->h = src0->h + (arc_measures.h * interp_perc);
 	dest0->b = src0->b + (arc_measures.b * interp_perc);
 }
+
+std::ostream& operator<<(std::ostream& os, const vec3d& vec)
+{
+	os << "vec3d<" << vec.xyz.x << ", " << vec.xyz.y << ", " << vec.xyz.z << ")";
```

Comment:
This is going to show `vec3d<0, 0, 0)`.  Probably should pick either `<>` or `()`.

Reply:
Fixed.